intern/convenience/array.py

```diff
@@ -222,6 +392,30 @@ def bulk_update(self, items: dict):
     def bulk_delete(self, keys: list):
         return self._remote.delete_metadata(self._resource, keys)
 
+
+def _infer_volume_provider(channel: Union[ChannelResource, str, Tuple]):
```

Comment:
When the channel is in a tuple, what does that look like?

Reply:
At least internally, that means `[collection, experiment, channel]`, of type `Tuple[str, str, str]`. (We support that as a legacy option for passing them as `*args` to a Resource constructor)